SensorCode/SensorState.py

```diff
@@ -48,7 +117,16 @@ def update(self) -> None:
         """
         Update function to read the serial lines and update the sensor state
         """
-        pass
+        self.update_terabee()
```

Comment:
you return terabee data, but never set self.terabee_top, self.terabee_mid ... equal to the returned data

Reply:
the function update_terabee() doesn't return anything but instead just sets self.terabee_top, self.terabee_mid to their respective values